doc/modules/classes.rst

```diff
@@ -903,7 +903,7 @@ details.
 
    metrics.adjusted_mutual_info_score
    metrics.adjusted_rand_score
-   metrics.calinski_harabaz_score
```

Comment:
There's a list of deprecated functions and classes at the bottom. Can you move the old one there?